src/reporting/clients/IReportsClient.ts

```diff
@@ -0,0 +1,149 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import type { Report, ReportCreate, ReportMapping, ReportMappingCreate, ReportUpdate } from "../interfaces/Reports";
+import type { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IReportsClient{
+  /**
+   * Gets all Reports within the context of a Project. This method returns the full list of reports.
+   * @param {string} projectId The Project Id.
```

Comment:
Change all Project Id references to iTwin Id in documentation and code.

Reply:
changed all project Id's to iTwin Id's everywhere except for the utils folder for integration tests

Reply:
reverted changes due to compatibility issues with backend